.eslintrc.cjs

```diff
@@ -1,21 +1,30 @@
 module.exports = {
+  root: true,
   env: {
     es2021: true,
     node: true,
     'jest/globals': true,
   },
-  extends: ['eslint:recommended', 'plugin:n/recommended'],
+  extends: ['eslint:recommended', 'plugin:n/recommended', 'plugin:@typescript-eslint/recommended'],
   parser: '@typescript-eslint/parser',
-  plugins: ['jest', '@typescript-eslint'],
-  overrides: [],
+  overrides: [
+    {
+      files: ['test/**'],
+      plugins: ['jest'],
+      extends: ['plugin:jest/recommended'],
+      rules: {},
+    },
+  ],
   parserOptions: {
     ecmaVersion: 'latest',
     sourceType: 'module',
   },
   rules: {
-    'n/no-missing-import': 'off',
```

Comment:
as mentioned elsewhere, let's bring this back since TS has us covered and does a better job than eslint